src/styles.js

```diff
@@ -43,6 +43,11 @@ color.propTypes = {
   ...bgColor.propTypes,
 }
 
+export const opacity = responsiveStyle({
```

Comment:
What is the use-case for responsive opacity? I've never used opacity responsively, so forgive me if this is a silly question

Reply:
Don't have a good use case but seems like it could be used as much as responsive color or border would be. Not really sure what the criteria should be for a responsive style vs a regular style.